naga/src/valid/expression.rs

```diff
@@ -1332,12 +1334,32 @@ impl super::Validator {
                             _ => return Err(ExpressionError::InvalidArgumentType(fun, 0, arg)),
                         }
                     }
-                    Mf::CountTrailingZeros
-                    | Mf::CountLeadingZeros
```

Comment:
`CountTrailingZeros` with non 32 bit types should also not be allowed.

Reply:
I did not find any spec saying they are not supported for 64 bit operands, and it does work fine in my tests. Check out the two pages linked in #5276

Reply:
The backend uses `FindILsb` for `CountTrailingZeros` which only works with 32 bit types.